build.gradle

```diff
@@ -13,6 +13,7 @@ sourceCompatibility = 1.8
 targetCompatibility = 1.8
 
 repositories {
+    mavenCentral()
```

Comment:
Do we need this?

Reply:
I thought the build had failed when I didn't include this but it seems to work fine now. I'll remove it.